package.json

```diff
@@ -28,9 +28,10 @@
     "react-redux": "^7.2.6",
     "react-router-dom": "^5.2.0",
     "react-router-transition": "^2.1.0",
-    "react-scripts": "5.0.0",
+    "react-scripts": "^5.0.0",
     "react-spinners": "^0.11.0",
     "sweetalert2": "^11.3.10",
+    "xxxx": "^3.0.4",
```

Comment:
Ésto está bien?

Reply:
tenia conflictos con una dependencia y no podia hacer funcionar la app.. Elimine la carpete de node modules y el package-lock.json y ya directamente no pude hacer npm install.. estuve buscando un poco y encontre esto npm add xxxx --legacy-peer-deps que me ayudo a solucionar todos los problemas

Reply:
Fijate que esa lib está apuntada más que nada a angular. No sé si en tu maquina tenes librerias globales de angular, pero no deberia ser necesario tener ésta dependencia

Reply:
Sumado a que el mismo repositorio dice:
_THIS REPOSITORY HAS BEEN ARCHIVED SINCE IT NO LONGER REFLECTS THE CURRENT STATE OF ANGUAR UNIVERSAL PROJECT GENERATED BY ANGULAR CLI. PLEASE SEE https://angular.io/guide/universal ON HOW TO SETUP AN UNIVERSAL PROJECT USING ANGULAR CLI._

Idealmente, no sumar libs que no están siendo mantenidas